source/READ.md

```diff
@@ -0,0 +1,43 @@
+
+# How to deploy the solution
+
+## Regions
+
+  [regions](../docs/en/regions.md)
+
+## Quick deployment
+
+  [deployment](../docs/en/deployment.md)
+
+## Deploy from source
+
+Deploy solution to your AWS account by [AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/home.html).
+### Prerequisites
+
+- An AWS account
+- Configure [credential of aws cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html)
+- Install Node.js LTS version, such as 12.x
+- Install Docker Engine
+- Install the dependencies of solution via executing command `npm install`
+- Initialize the CDK toolkit stack into AWS environment(only for deploying via AWS CDK first time, [doc](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install))
+- Enable Amazon Quicksight account in your AWS account
```

Comment:
pls link to doc how to configure QuickSight user

Reply:
add doc link